src/dsp_tools/commands/excel2json/project.py

```diff
@@ -59,20 +60,22 @@ def excel2json(
 def _validate_folder_structure_get_filenames(data_model_files: str) -> tuple[list[Path], list[Path]]:
     if not Path(data_model_files).is_dir():
         raise UserError(f"ERROR: {data_model_files} is not a directory.")
-    folder = [x for x in Path(data_model_files).glob("*") if _non_hidden(x)]
-    processed_files = []
-    onto_folders, processed_onto = _get_and_validate_onto_folder(Path(data_model_files), folder)
-    processed_files.extend(processed_onto)
-    listfolder, processed_lists = _get_validate_list_folder(data_model_files, folder)
-    processed_files.extend(processed_lists)
-    if len(onto_folders) + len(listfolder) != len(folder):
+    sub_folders = [x for x in Path(data_model_files).glob("*") if _non_hidden(x) and x.is_dir()]
+    files_to_process = []
+    onto_folder_content, onto_files_to_process = _get_and_validate_onto_folder(Path(data_model_files), sub_folders)
+    files_to_process.extend(onto_files_to_process)
+    listfolder_content, lists_to_process = _get_validate_list_folder(data_model_files, sub_folders)
```

Comment:
listfolder_content: This one is awkward: It is a list containing only 1 element, namely the list folder. This is misleading per se and cannot be improved just by renaming. So let's keep it for today, in order not to make this PR too big 😃

Reply:
I just changed it back

Reply:
Okay, you already improved it. Yes, now it's better understandable for me than before. Thank you :-)